doc/camera_and_viewport.md

```diff
@@ -0,0 +1,45 @@
+# Camera and Viewport
+
+When rendering on Flutter, the regular coordinate space used are logical pixels. That means one pixel for Flutter is already not necessarily one real pixel on the device, because of the [device's pixel ratio](https://api.flutter.dev/flutter/widgets/MediaQueryData/devicePixelRatio.html). When it gets to the Flame level, we always consider the most fundamental level to be logical pixels, so all the device specific complexity is abstracted away.
+
+However, that still leaves you with arbitrarily shaped and sized screens. And it's very likely that your game has some sort of game world with an intrinsic coordinate system that does not map to screen coordinates. Flame adds two distinct concepts to help transform coordinate spaces. For the former, we have the Viewport class. And for the later, we have the Camera class.
+
+## Viewport
+
+The Viewport is an attempt to unify multiple screen (or, rather, game widget) sizes into a single configuration for your game by translating and resizing the canvas.
+
+The `Viewport` interface has multiple implementations and can be used from scratch on your `Game` or, if you are using `BaseGame` instead, it's already built-in (with a default no-op viewport).
+
+These are the viewports available to pick from (or you can implement the interface yourself to suit your needs):
+
+ * `DefaultViewport`: this is the no-op viewport that is associated by default with any `BaseGame`.
+ * `FixedResolutionViewport`: this viewport transforms your Canvas so that, from the game perspective, the dimensions are always set to a fixed pre-defined value. This means it will scale the game as much as possible and add black bars if needed.
+ * More viewports are planned!
```

Comment:
Not sure we need this line her

```suggestion
```

Reply:
I found it weird to have a list with only two, but let me remove